src/plugins/kibana_react/public/code_editor/code_editor.tsx

```diff
@@ -154,6 +154,16 @@ export class CodeEditor extends React.Component<Props, {}> {
 
     this._editor = editor;
 
+    editor.addCommand(
+      monaco.KeyCode.Escape,
+      () => {
+        if (document.activeElement instanceof HTMLElement) {
+          document.activeElement.blur();
```

Comment:
When you `.blur()` you're removing focus from the entire page which puts it back on the `<body>`. Some browsers+assistive tech combinations will figure out that something went wrong and try to put you back near where you were but others won't – either way, relying on that functionality is pretty brittle.

Instead of simply removing focus, we need to move it to another element or in some other way allow tabbing to move focus to the next tabbable element.